src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -248,6 +251,8 @@ function validateParams(vmParams) {
     }
     if (vmParams.userPassword && !vmParams.userLogin) {
         validationFailed.userLogin = _("User login must not be empty when user password is set");
+    } else if (vmParams.sshKeys.length > 0 && !vmParams.userLogin) {
+        validationFailed.userLogin = _("User login must not be empty when SSH keys are set");
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1150-20230718-074723-da6d3da1-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)